app/src/main/java/org/fossasia/openevent/app/data/network/EventService.java

```diff
@@ -102,4 +103,7 @@
 
     @POST("faqs")
     Observable<Faq> postFaq(@Body Faq faq);
+
+    @GET("events/{id}/feedbacks")
```

Comment:
See the URL for FAQs, tickets, attendees, etc and it will be corrected automatically